mne_bids_pipeline/config.py

```diff
@@ -226,6 +226,14 @@ class ArbitraryContrast(TypedDict):
 automatically.
 """
 
+process_rest: bool = True
```

Comment:
Looks good but I would not process rest by default. For cam-can that would mean a lot of computations that I typically don't need when looking at task data.

Reply:
I think Cam-can is an exception here really. To argue the default should be False means arguing that the default behavior should be "only process a subset of the data, not as much as possible". Are you sure this is the best choice for most datasets / principle of least surprise for people?

Reply:
To give context, datasets I work on with both task and rest data have 30 minutes of task and 5 of rest. And I always want them processed jointly for analysis. I expect more people to be in this position than in the position "I have access to both types of data but want to leave unprocessed the rest data".

Thinking about in the future potentially adding _05_rest_envelope_correlations.py or something, it would also be nice to have this on by default :)

But you work with more datasets and researchers than me, so if you're not convinced by this I'll change it!

Reply:
@agramfort from a chat with @hoechenberger he's on board with the "process all data until told otherwise" idea, so I'm going to keep it that way and merge for now. But let me know if you disagree with this and I can open a follow-up PR to change the default back